### PR TITLE
A more accurate description of the tutorial on Monday.

### PR DESCRIPTION
**Pull request by arokem**
yip

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Brain Hacking 101 Course for the 2015 HBM Hackathon in Honolulu Hawaii
 
 ### Monday 
 
-- Configuring your system for scientific computing
+At 1 PM, 2PM, and 3 PM (50 minutes)
```
Comment (bcipolli) [inline README.md]: If the same tutorial will be given at this time, I might say "Repeating at 1pm...." to be clear.

**Comment by nicholsn** [comment]
thanks!

File: README.md (hunk 1 above)
Comment (eickenberg) [inline README.md]: +1 :)

**Comment by eickenberg** [comment]
LGTM +1 for big merge :)

**Comment by arokem** [comment]
Are you sure? Shouldn't we wait for the CI to test this?

**Comment by nicholsn** [comment]
=)

**Comment by eickenberg** [comment]
Argh, true! I hope the space inconsistency  `1 PM, 2PM, 3 PM` doesn't break the conference!

**Pull request merged**